geode-redis/src/main/java/org/apache/geode/redis/internal/executor/set/StripedExecutor.java

```diff
@@ -35,9 +34,7 @@
    *
    * @param stripeId defines the "stripe"
    * @param callable the unit of work to do sequentially. May be called after run returns.
-   * @param resultConsumer is given the result of the callable.
    */
-  public <T> void execute(Object stripeId,
-      Callable<T> callable,
-      Consumer<T> resultConsumer);
+  public <T> T execute(Object stripeId,
```

Comment:
remove the javadocs on this method that talk about "the result of..."

Reply:
done.